src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -209,9 +210,13 @@ def app_create(cmd, client, resource_group, service, name,
         instance_count=instance_count,
         environment_variables=env,
         jvm_options=jvm_options,
-        runtime_version=runtime_version,)
+        net_core_main_entry_path=None,
+        runtime_version=runtime_version)
+
+    file_type = "NetCoreZip" if runtime_version == AppPlatformEnums.RuntimeVersion.net_core_31 else "Jar"
```

Comment:
CaseIgnore comparasion?

NIT: All the validation, default value set you can put to [validator](https://github.com/Azure/azure-cli-extensions/blob/master/src/spring-cloud/azext_spring_cloud/_validators.py) and set `namespace.runtime_version=XXX`, thus can skip the file_type setting here, `_get_upload_local_file()` method as well.

Reply:
don't quite understand your comment, let's sync it offline.